### DISCUSSION TITLE
Fix

### DISCUSSION THREAD

**Discussion by darksider30** [JustOptimize/return-ShowHiddenChannels#189] [Q&A]
Hello 
Discord has been updated. Unfortunately, ShowHiddenChannels does not work. can you help me?
![shc](https://github.com/JustOptimize/return-ShowHiddenChannels/assets/173217976/65727c62-a3e2-4c54-a688-b98655d66893)

**Comment by JustOptimize** [comment]
This is not the issues section and #188 already exists, closing
